modules/nw-multus-ipvlan-object.adoc

```diff
@@ -6,6 +6,8 @@
 // https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_networking/getting-started-with-ipvlan_configuring-and-managing-networking#ipvlan-overview_getting-started-with-ipvlan
 :_content-type: REFERENCE
 
+:_module-type: CONCEPT
```

Comment:
Sorry, I'm not familiar with `_module-type`; is it required for TELCO docs? But either way, the value `CONCEPT` here seems at odds with the `_content-type: REFERENCE` above it.